rel-eng/atomic-enterprise-allinone.service

```diff
@@ -0,0 +1,22 @@
+[Unit]
+Description=Atomic Enterprise Master All-In-One
+Documentation=https://github.com/openshift/origin
+After=network.target
+After=etcd.service
+Before=atomic-enterprise-sdn-node.service
+Before=atomic-enterprise-sdn-master.service
+Requires=network.target
+
+[Service]
+Type=notify
+EnvironmentFile=/etc/sysconfig/atomic-enterprise-allinone
+Environment=GOTRACEBACK=crash
+ExecStart=/usr/bin/atomic-enterprise start --master-config=${MASTER_CONFIG_FILE} --node-config=${NODE_CONFIG_FILE} $OPTIONS
```

Comment:
We already set GOMAXPROCS in all the binaries to number of cores by default
(if you haven't set GOMAXPROCS)

On Mon, Aug 10, 2015 at 4:18 PM, Timothy St. Clair <notifications@github.com
> wrote:

> In rel-eng/atomic-enterprise-allinone.service
> <https://github.com/openshift/origin/pull/3955#discussion_r36678966>:
>
> > @@ -0,0 +1,22 @@
> > +[Unit]
> > +Description=Atomic Enterprise Master All-In-One
> > +Documentation=https://github.com/openshift/origin
> > +After=network.target
> > +After=etcd.service
> > +Before=atomic-enterprise-sdn-node.service
> > +Before=atomic-enterprise-sdn-master.service
> > +Requires=network.target
> > +
> > +[Service]
> > +Type=notify
> > +EnvironmentFile=/etc/sysconfig/atomic-enterprise-allinone
> > +Environment=GOTRACEBACK=crash
> > +ExecStart=/usr/bin/atomic-enterprise start --master-config=${MASTER_CONFIG_FILE} --node-config=${NODE_CONFIG_FILE} $OPTIONS
>
> GOMAXPROCS ? or is someone setting that internally?
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/openshift/origin/pull/3955/files#r36678966>.
>



-- 
Clayton Coleman | Lead Engineer, OpenShift